### PR TITLE
OracleCorporation.Java(TM)PlatformSE19.0.2 version 19.0.2.0

### PR DESCRIPTION
**Pull request by palladinoandrea**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/108620)

### CODE REVIEW

**Review by russellbanks — CHANGES_REQUESTED** [review]
Oracle JDK is already present under `Oracle.JDK`

File: manifests/o/OracleCorporation/Java(TM)PlatformSE19/0/2/19.0.2.0/OracleCorporation.Java(TM)PlatformSE19.0.2.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: OracleCorporation.Java(TM)PlatformSE19.0.2
```
Comment (stephengillie) [inline manifests/o/OracleCorporation/Java(TM)PlatformSE19/0/2/19.0.2.0/OracleCorporation.Java(TM)PlatformSE19.0.2.installer.yaml:4]:
```suggestion
PackageIdentifier: Oracle.JDK.19
```

File: manifests/o/OracleCorporation/Java(TM)PlatformSE19/0/2/19.0.2.0/OracleCorporation.Java(TM)PlatformSE19.0.2.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: OracleCorporation.Java(TM)PlatformSE19.0.2
```
Comment (stephengillie) [inline manifests/o/OracleCorporation/Java(TM)PlatformSE19/0/2/19.0.2.0/OracleCorporation.Java(TM)PlatformSE19.0.2.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Oracle.JDK.19
```

File: manifests/o/OracleCorporation/Java(TM)PlatformSE19/0/2/19.0.2.0/OracleCorporation.Java(TM)PlatformSE19.0.2.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: OracleCorporation.Java(TM)PlatformSE19.0.2
```
Comment (stephengillie) [inline manifests/o/OracleCorporation/Java(TM)PlatformSE19/0/2/19.0.2.0/OracleCorporation.Java(TM)PlatformSE19.0.2.yaml:4]:
```suggestion
PackageIdentifier: Oracle.JDK.19
```

File: manifests/o/OracleCorporation/Java(TM)PlatformSE19/0/2/19.0.2.0/OracleCorporation.Java(TM)PlatformSE19.0.2.locale.en-US.yaml (hunk 2 above)
Comment (stephengillie) [reply manifests/o/OracleCorporation/Java(TM)PlatformSE19/0/2/19.0.2.0/OracleCorporation.Java(TM)PlatformSE19.0.2.locale.en-US.yaml:4]: Please update file name and path to match this change.

File: manifests/o/OracleCorporation/Java(TM)PlatformSE19/0/2/19.0.2.0/OracleCorporation.Java(TM)PlatformSE19.0.2.installer.yaml (hunk 1 above)
Comment (stephengillie) [reply manifests/o/OracleCorporation/Java(TM)PlatformSE19/0/2/19.0.2.0/OracleCorporation.Java(TM)PlatformSE19.0.2.installer.yaml:4]: Please update file name and path to match this change.

**Comment by stephengillie** [comment]
Readds #105298

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

File: manifests/o/OracleCorporation/Java(TM)PlatformSE19/0/2/19.0.2.0/OracleCorporation.Java(TM)PlatformSE19.0.2.yaml (hunk 3 above)
Comment (stephengillie) [reply manifests/o/OracleCorporation/Java(TM)PlatformSE19/0/2/19.0.2.0/OracleCorporation.Java(TM)PlatformSE19.0.2.yaml:4]: Please update file name and path to match this change.

File: manifests/o/OracleCorporation/Java(TM)PlatformSE19/0/2/19.0.2.0/OracleCorporation.Java(TM)PlatformSE19.0.2.locale.en-US.yaml (hunk 2 above)
Comment (stephengillie) [reply manifests/o/OracleCorporation/Java(TM)PlatformSE19/0/2/19.0.2.0/OracleCorporation.Java(TM)PlatformSE19.0.2.locale.en-US.yaml:4]: Please update file name and path to match this change.

File: manifests/o/OracleCorporation/Java(TM)PlatformSE19/0/2/19.0.2.0/OracleCorporation.Java(TM)PlatformSE19.0.2.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: OracleCorporation.Java(TM)PlatformSE19.0.2
+PackageVersion: 19.0.2.0
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://download.oracle.com/java/19/archive/jdk-19.0.2_windows-x64_bin.exe
+  InstallerSha256: 17412499045FA442059AAB83F5480B9808F7C2DD94FAE9175811774D9E859B7C
```
Comment (stephengillie) [inline manifests/o/OracleCorporation/Java(TM)PlatformSE19/0/2/19.0.2.0/OracleCorporation.Java(TM)PlatformSE19.0.2.installer.yaml:10]:
```suggestion
  InstallerSha256: 17412499045FA442059AAB83F5480B9808F7C2DD94FAE9175811774D9E859B7C
  InstallerSwitches:
    Silent: /s
```

**Comment by ItzLevvie** [comment]
Close with reason: Stale pull request;

**Pull request closed without merging**